### PR TITLE
Split DataServlet into two seperate files

### PR DESCRIPTION
**Pull request by lukasberglund**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java [hunk 1]
```diff
@@ -16,23 +16,22 @@
...
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/comments")
-public class DataServlet extends HttpServlet {
+@WebServlet("/comments/list")
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java:33]: in google's API naming convention, we would use "/comments".

File: portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comments/new")
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java:26]: Up to you if you want to do it this way, but google's convention is "/comment" here and we'd use the http method to differentiate between the actions. doPost would map to "create", doGet == get, doDelete == delete, doPut == update

Comment (lukasberglund) [reply portfolio/src/main/java/com/google/sps/servlets/NewCommentServlet.java]: Cool. I'll go with that, I want to be Googley after all.

File: portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java [hunk 3]
```diff
@@ -16,23 +16,22 @@
...
-import java.util.ArrayList;
-import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java:33]: can you update this comment

File: portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/comment")
```
Comment (prabenzom) [inline portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java:26]: can you add a comment explaining what this servlet does?

**Pull request merged**